code-style/git/README.md

```diff
@@ -117,7 +117,8 @@ Find the [allowed types above](#Allowed-Values-for-Types).
 
 An [imperative tone][365] is also helpful in conveying what a commit does,
 rather than what it did. For example, use **change**, not _changed_ or
-_changes_.
+_changes_. To put it another way: a good subject should complete this sentence:
+["when applied, this commit will..."][drew-devault-link]
```

Comment:
I think this type of prompt is really helpful. I'm trying to put myself in the committer's shoes and see how this changes my instincts. 

> when applied, this commit will...
* `Adds dependabot config`
* `adds XYZController`

I'm not sure `this commit will` puts my focus on what the commit changes.

If we switched the wording a little to `when this commit is applied, ____`
* `Dependabot is run every day`
* `XYZController handles /xyz/`
* `Users can see their XYZs`

It's not perfect either, but I felt like it took the focus away from what the commit does, which is clear in the diff.

Reply:
I like either prompt.
Is the issue that this prompt necessitates filling out the body in a useful way? Do we need to add a prompt for the body too?

Reply:
I've been using the above `when this commit is applied, ___ ` to help me write better commit subjects. I think using `this commit will` makes you use a verb to start the subject like "add", "fix", "remove" which I can see in @cromwellryan's opinion could state the obvious if you looked at the code to see what was added or removed. Using `then this commit is applied` can help you focus more on the impact of removing or adding code like now that we added dependabot config, dependabot will run every week. I'm sure there are times when using a verb like "add" or "remove" would be sufficient in some cases too like when we remove peoples ssh keys from a repo.